packages/camera/camera/lib/src/camera_image.dart

```diff
@@ -125,4 +128,15 @@ class CameraImage {
   ///
   /// The number of planes is determined by the format of the image.
   final List<Plane> planes;
+
+  /// The aperture settings for this image.
+  ///
+  /// Represented as an f-stop value.
+  final double lensAperture;
+
+  /// The sensor exposure time for this image in nanoseconds.
+  final int sensorExposureTime;
+
+  /// The sensor sensitivity in standard ISO arithmetic units.
+  final double sensorSensitivity;
```

Comment:
Shouldn't these be nullable?

From what I can see (I might be missing something) on Android these values can potentially be `null`. When the JAVA `CameraCaptureProperties` class is instantiated the initial values will all be `null`, now if the `CaptureResult` object also returns a `null` value the properties are not set which still results in a `null` value communicated to Flutter.

Reply:
They should definitely be nullable yes, will make a fix.